src/main/java/fr/minuskube/inv/InventoryManager.java

```diff
@@ -129,12 +129,12 @@ public void onInventoryClick(InventoryClickEvent e) {
             if(!inventories.containsKey(p))
                 return;
 
-            if( e.getAction() == InventoryAction.COLLECT_TO_CURSOR ||
-                e.getAction() == InventoryAction.MOVE_TO_OTHER_INVENTORY ||
-                e.getAction() == InventoryAction.NOTHING) {
-
+            if(e.getAction() == InventoryAction.COLLECT_TO_CURSOR) {
+                e.setCancelled(true);
```

Comment:
You should add `return;` like the old. If you don't add, 149. line will work, so it may cause bugs.